inst/@symfun/minus.m

```diff
@@ -74,14 +74,14 @@
 %! f(x) = x^2;
 %! g = x^2;
 %! s = warning('off', 'OctSymPy:sym:arithmetic:workaround42735');
-%! h = x - f;  assert(isa(h, 'symfun') && isequal(h.sym, x - g))
-%! h = x + f;  assert(isa(h, 'symfun') && isequal(h.sym, x + g))
-%! h = x * f;  assert(isa(h, 'symfun') && isequal(h.sym, x * g))
-%! h = x / f;  assert(isa(h, 'symfun') && isequal(h.sym, x / g))
-%! h = x ^ f;  assert(isa(h, 'symfun') && isequal(h.sym, x ^ g))
-%! h = x .* f; assert(isa(h, 'symfun') && isequal(h.sym, x .* g))
-%! h = x ./ f; assert(isa(h, 'symfun') && isequal(h.sym, x ./ g))
-%! h = x .^ f; assert(isa(h, 'symfun') && isequal(h.sym, x .^ g))
+%! h = f - x;  assert(isa(h, 'symfun') && isequal(h.symbol, g - x))
+%! h = f + x;  assert(isa(h, 'symfun') && isequal(h.symbol, g + x))
+%! h = f * x;  assert(isa(h, 'symfun') && isequal(h.symbol, g * x))
+%! h = f / x;  assert(isa(h, 'symfun') && isequal(h.symbol, g / x))
+%! h = f ^ x;  assert(isa(h, 'symfun') && isequal(h.symbol, g ^ x))
+%! h = f .* x; assert(isa(h, 'symfun') && isequal(h.symbol, g .* x))
+%! h = f ./ x; assert(isa(h, 'symfun') && isequal(h.symbol, g ./ x))
+%! h = f .^ x; assert(isa(h, 'symfun') && isequal(h.symbol, g .^ x))
```

Comment:
Well, in minus atleast, we are calling minus on the `symbol` of symfun and the sym and then converting the result to symfun. If it's ok I can add conditions in `minus` of `sym` so that it converts the result to `symfun` if either of the operands is `symfun`.. Is that ok? Or should I mark it as xtest?